tinygrad/opsgpu.py

```diff
@@ -334,8 +334,10 @@ def forward(ctx, x, shape):
   @staticmethod
   def backward(ctx, grad_output):
     in_shape, = ctx.saved_tensors
-    grad_output.shape = in_shape
-    return grad_output
+    ret = new_buffer(in_shape)
```

Comment:
Hmm, this isn't what bumpy reshape does

Reply:
Also, what is new_buffer? Isn't it buffer_new?

Reply:
Reverting this.... tried to get rid of MNIST error #69

Reply:
If you want a copy, just the unary op is fine.

Reply:
Yeah not sure if we should copy?

Reply:
Tensors on GPU maintain an array on cpu (.data) which determines the shape. If you want to change tensor shape you need to reshape that array or we need to maintain the shape as a separate property